packages/point-of-sale/src/payments/service.test.ts

```diff
@@ -58,12 +58,13 @@ describe('createPaymentService', () => {
     expect(result).toEqual({ id: uuid, url: expectedUrl })
     expect(mockApolloClient.mutate).toHaveBeenCalledWith(
       expect.objectContaining({
-        variables: {
+        variables: expect.objectContaining({
           walletAddressId,
           incomingAmount,
           idempotencyKey: expect.any(String),
-          isCardPayment: true
-        }
+          isCardPayment: true,
+          expiresAt: expect.any(String)
```

Comment:
we can mock Date.now and specify the actual value for `expiresAt` we expect here